rerun_py/rerun_bindings/rerun_bindings.pyi

```diff
@@ -1284,7 +1283,7 @@ class Entry:
         """The entry's name."""
 
     @property
-    def catalog(self) -> CatalogClient:
+    def catalog(self) -> CatalogClientInternal:
```

Comment:
That's correct, but also extremely broken. We shouldn't be exposing or returning the internal catalog here. I guess the best way to deal with this correctly is to roll the wrapper/internal pattern to entry.

```suggestion
    # TODO(RR-2938): this should return `CatalogClient`
    def catalog(self) -> CatalogClientInternal:
```